src/components/Calculator.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import './calculator.css';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
- [ ] Please you don't need to put props as a parameter `(props) `props is a built-in method in class-based components you can say `this. props` and use the props you need
- [ ] in case you need this hint to use this.setState(); inside a method in your class you have to use the bind method in your constructor 
class Calculator extends Component {
constructor(){
super();
this.state = {// initial state}
this.methodname = this.methodname.bind(this); // bind is a 
// method which means I 
// I want the `this` keyword to represent the class not the function
};
methodname(){
this.setState({newState})}
render(){
return (JSX code);